src/Morphir/JsonSchema/Backend.elm

```diff
@@ -105,55 +105,96 @@ extractTypes modName definition =
             )
 
 
-mapTypeDefinition : TypeName -> Type.Definition ta -> Maybe ( TypeName, SchemaType )
-mapTypeDefinition typeName definition =
+mapTypeDefinition : ( Path, Name ) -> Type.Definition ta -> List ( TypeName, SchemaType )
```

Comment:
Can you please change the type to `Result` and make sure that we fail when we cannot map a type instead of just ignoring it?